### ISSUE TITLE
[Adaptive Card iOS - LongLabels.json]: No name is defined for the button present on the Long Label card.

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#192]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.4.0-beta.24.5.30.1)

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
Device: iPhone 11
iOS: 17.5.1
Application: Adaptive Cards
App Version: Version 1.0 (2.4.0-beta.24.5.30.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.3" control and select it.
5. Navigate to 'Test' control and activate it.
6. Navigate to 'LongLabels.Jsons' and activate it.
7. The Card will open.
8. Perform right swipe and navigate to button control and observe the issue.

**Actual Result:**
There is no name defined for button present under the card due to this when the screen reader focus lands on it then Voice Over is conveying as 'Button' only.

**Expected Result:**
Name should be defined for the button so that when the voice over focus lands on it then screen reader should announce something like **'Cat button.'**

**User Impact:**
It will impact the screen reader user as they will not be able to know the name of the button control and due to this they will not able to know on activating it what action will be performed.

**WCAG Reference:**
https://www.w3.org/TR/WCAG21/#name-role-value

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/82f9537e-1975-49ab-9449-96c4f3efa634

### Card JSON

```JSON
NA
```

**Comment by GabrielMedAlv** [comment]
PR fix link: https://github.com/microsoft/Teams-AdaptiveCards-Mobile/pull/243
